waterlog/src/Components/DailyCostsReports.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { Line, defaults } from 'react-chartjs-2';
+
+const DailyCostsReports = (props) => { 
+  var labelX = props.props.dataPoints.map(a => (new Date(a.x).getHours() + ":00"));
+  var dataY = props.props.dataPoints.map(a => a.y);
+  var today = new Date(Date.now());
+  var data = {
+    labels: labelX,
+    datasets: [
+      {
+        label: 'Rands',
+        data: dataY,
+        fill: true,
+        borderColor: 'red',
+        backgroundColor: 'rgba(255,0,0,0.4)'
```

Comment:
I know you got a variables file in your project, please use them in such places:
e.g
~backgroundColor: 'rgba(255,0,0,0.4)'~
but
`backgroundColor: $some-colour`

where;  $some-colour: 'rgba(255,0,0,0.4)';

Reply:
Thanks I will consider this.

Reply:
this was not a suggestion for the next PR but ...